URL Encode ApplicationID/DeviceID

stefaneberl opened the pull request:
In addition, sometimes the underscore character _
is reserved in Dynatrace/AppMon, therefore we
need to URL-encode this character as well.

stefaneberl committed:
URL Encode ApplicationID/DeviceID

In addition, sometimes the underscore character _
is reserved in Dynatrace/AppMon, therefore we
need to URL-encode this character as well.

y0chanan commented on src/openkit-shared/Util/IEnumerableExtensions.cs:
```diff
@@ -0,0 +1,37 @@
+﻿//
+// Copyright 2018 Dynatrace LLC
```
duplicate file ?
src/openkit-shared/Util/IEnumerableExtensions.cs
src/openkit-shared/Core/Util/IEnumerableExtensions.cs

stefaneberl replied:
Good finding, thanks!
Indeed it's a duplicate and `src/openkit-shared/Util/IEnumerableExtensions.cs` will remain.

stefaneberl committed:
Fix findings of code review

IEnumerableExtensions was added twice, but only
the one in Util/ folder should be there.
This has been fixed accordingly.
